transformer_engine/tensorflow/fp8.py

```diff
@@ -69,7 +69,7 @@ def get_default_fp8_recipe():
 def fp8_autocast(
     enabled: bool = False,
     fp8_recipe: Optional[DelayedScaling] = None,
-) -> None:
+) -> Generator[None, Any, None]:
```

Comment:
We don't intend to use `send` with this generator:
```suggestion
) -> Generator[None, None, None]:
```